lib/csv-stringifiers/abstract.js

```diff
@@ -16,10 +16,12 @@ class AbstractCsvStringifier {
     }
 
     stringifyRecords(records) {
-        const csvLines = records
```

Comment:
Doesn't it also work if we just change `records` to `Array.from(records)`?

Reply:
It would be functional, but it would be very inefficient.